mdbenchmark/mdengines/namd.py

```diff
@@ -0,0 +1,165 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDBenchmark
+# Copyright (c) 2017 Max Linke & Michael Gecht and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# MDBenchmark is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# MDBenchmark is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with MDBenchmark.  If not, see <http://www.gnu.org/licenses/>.
+import os
+import warnings
+from glob import glob
+from shutil import copyfile
+
+import click
+import datreant.core as dtr
+import mdsynthesis as mds
+import numpy as np
+from six import string_types
+
+
+def parse_ns_day(fh):
+    """parse nanoseconds per day from a NAMD log file
+
+    Parameters
+    ----------
+    fh : str / filehandle
+        filename or string of log file to read
+
+    Returns
+    -------
+    float
+        nanoseconds per day
+    """
+    lines = fh.readlines()
+
+    for line in lines:
+        if 'Benchmark time' in line:
+            return 1 / float(line.split()[7])
+
+    return np.nan
+
+
+def parse_ncores(fh):
+    """parse number of cores from a NAMD log file
+
+    Parameters
+    ----------
+    fh : str / filehandle
+        filename or string of log file to read
+
+    Returns
+    -------
+    float
+        number of cores job was run on
+    """
+    lines = fh.readlines()
+
+    for line in lines:
+        if 'Benchmark time' in line:
+            return int(line.split()[3])
+
+    return np.nan
+
+
+def analyze_run(sim):
+    """
+    Analyze Performance data of a NAMD simulation
+    """
+    ns_day = np.nan
+    ncores = np.nan
+
+    # search all output files and ignore GROMACS backup files
+    output_files = glob(os.path.join(sim.relpath, '[!#]*out*'))
+    if output_files:
+        with open(output_files[0]) as fh:
+            ns_day = parse_ns_day(fh)
+            fh.seek(0)
+            ncores = parse_ncores(fh)
+
+    # Backward compatibility to previously created mdbenchmark systems
```

Comment:
We probably do not need this here. We added it to the GROMACS module, because we added more categories to your `Sim` objects at some point. All new Sim objects for NAMD will include that category `time`.